adapters/appush/appush_test.go

```diff
@@ -0,0 +1,20 @@
+package appush
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderAppush, config.Adapter{
+		Endpoint: "http://sa-lb.deliverimp.com/pserver"}, config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})
```

Comment:
Please replace endpoint url to something like "test.com", so it's more obvious it's test. It will require changes in json tests.